### PR TITLE
Feature-Bright:send-assessment-email-api

### PR DESCRIPTION
**Pull request by Brightlaz**
Created the api and followed all the instructions

### CODE REVIEW

**Review by edidee — APPROVED** [review]

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Please remove the comments

File: Developers/app/api/sendAssessmentEmail/route.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { NextRequest, NextResponse } from "next/server";
+import nodemailer from "nodemailer";
+
+export async function POST() {
+  try {
+    // Create a Nodemailer transporter
+    const transporter = nodemailer.createTransport({
+      service: process.env.EMAIL_SERVICE_PROVIDER, // e.g., 'Gmail', 'Yahoo', etc.
```
Comment (annonymousauthority) [inline Developers/app/api/sendAssessmentEmail/route.js:8]: You're going to need 
`NEXT_PUBLIC_[NAME_OF_VARIABLE]` for Next.js environment variable

File: Developers/app/api/sendAssessmentEmail/route.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { NextRequest, NextResponse } from "next/server";
+import nodemailer from "nodemailer";
+
+export async function POST() {
```
Comment (annonymousauthority) [inline Developers/app/api/sendAssessmentEmail/route.js:4]: It requires a parameter requests

File: Developers/app/api/sendAssessmentEmail/route.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    const transporter = nodemailer.createTransport({
+      service: process.env.EMAIL_SERVICE_PROVIDER, // e.g., 'Gmail', 'Yahoo', etc.
+      auth: {
+        user: process.env.EMAIL,
+        pass: process.env.EMAIL_PASSWORD,
+      },
+    });
+
+    // Extract email data from the request body
+    const { to, subject, html } = NextRequest.body;
```
Comment (annonymousauthority) [inline Developers/app/api/sendAssessmentEmail/route.js:16]: I'll recommend more of the `await request.json()` request being the parameter argument which will be the body.

File: Developers/app/api/sendAssessmentEmail/route.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+    // Define the email message
+    const mailOptions = {
+      from: "your-email@example.com",
```
Comment (annonymousauthority) [inline Developers/app/api/sendAssessmentEmail/route.js:20]: This should be an environment variable called
`NEXT_PUBLIC_SENDER_EMAIL`

File: Developers/app/api/sendAssessmentEmail/route.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+      subject,
+      html,
+    };
+
+    // Send the email
+    await transporter.sendMail(mailOptions);
+
+    NextResponse.status(200).json({ message: "Email sent successfully" });
+  } catch (error) {
+    console.error("Error sending email:", error);
```
Comment (annonymousauthority) [inline Developers/app/api/sendAssessmentEmail/route.js:31]: CONSOLE.error can be removed

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
I'll like to believe this code wasn't copied from somewhere, which obviously it was.

Just work on the points mentioned.

File: Developers/app/api/sendAssessmentEmail/route.js (hunk 1 above)
Comment (Brightlaz) [reply Developers/app/api/sendAssessmentEmail/route.js:8]: Just checking the Next documentation now which confirms i called the environment variable the right way sir, 
Or if it not what you were referring to please direct me to the particular section

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Ok this works, just write a function that handles handlebar template configuration not inline.

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Ok this works, just write a function that handles handlebar template configuration not inline.

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Not configured for handlebars template

**Review by annonymousauthority — APPROVED** [review]
LG

**Pull request merged**